main.py

```diff
@@ -38,18 +40,23 @@ def get_stats() -> str:
     '''Gets API data and returns markdown progress'''
     data = requests.get(
         f"https://wakatime.com/api/v1/users/current/stats/last_7_days?api_key={waka_key}").json()
+
     try:
         lang_data = data['data']['languages']
     except KeyError:
         print("Please Add your Wakatime API Key to the Repository Secrets")
         sys.exit(1)
+
     data_list = []
+    lang_data = data['data']['languages']
```

Comment:
Should this be here? We're already initializing lang data in line 45 in the the try-except block. If we again use lang_data here. It might lead another `KeyError` if the API Call didn't work in the first place

Reply:
@joe733

Reply:
My bad, missed it when resolving conflicts.